File: python/minispec/simple.py

if __name__ == '__main__':
    import nose
    from must_methods import gself
    import os
    def setup():
        gself.ls = "ls"
    def test_must_equal():
        print "ok"
        print gself.ls
        'sdd',(1).must_equal(1)

    # support print in the test methods
    os.environ['NOSE_NOCAPTURE'] = '1'
    result = nose.runmodule()

File: js/node_test/simple.js
if (require.main === module) {
    var mm="中文";
    console.log(mm);

    var array = ['a', 'b', 'c'];

    for (var i in array) {
      console.log(array[i]);
    }
    console.dir({name:mm})
    console.log({name:mm})
}

